<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfficeView</class>
 <widget class="QWidget" name="OfficeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Office View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget1">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FirstName</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LastName</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Department</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Salary</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Update Salary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>Edit Record</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete Record</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewAttendanceButton">
       <property name="text">
        <string>Mark Attendance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="generatePaySlipButton">
       <property name="text">
        <string>Generate PayRoll</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
